coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/LayerResource.java

```
@@ -186,8 +191,11 @@ public Response createRasterLayer(
                 log.info("Raster layer create - about to addRasterLayer to geoserver with an EPSG of: " + EPSGcode);
                 
                 Service rasterService = GeoserverUtil.addRasterLayer(geoserverEndpoint, zipFileStream, newId, bbox, EPSGcode);
-               
-                services.add(rasterService);
+                if(null == rasterService) {
+                        throw new ServerErrorException("Unable to create a store and/or layer in GeoServer.", Status.INTERNAL_SERVER_ERROR);
```

This error handling was lacking before, thus masking errors happening via GeoServer. I'm open to suggestion on whether this should be implemented as it is currently (a null-check-on-return), or as a thrown-and-caught exception.